build_support/project.py

```diff
@@ -1,3 +1,3 @@
 from pathlib import Path
 
-PATH = Path(__file__).parent.parent.resolve()
+PATH = Path("../..")
```

Comment:
Thank you, Jordi!  I have removed use of **project.PATH** and replaced it with a locally-defined constant declared near the top of the file, so it would be easy to find it and change it if necessary.  If this a better solution in your view?

Reply:
Thank you so much for your review!  So — is the parent process invoked by the flake-check test script _not_ the project root directory?  I'd like to understand this a bit better.  I think option (b) seems to be a reasonable compromise between flexibility and simplicity.  Shall we discuss this at the next meeting?  (The good news is that I have my home environment finally set up correctly!)

Reply:
I think this is resolved.  I went with option a, as the simplest.  I don't think this restriction is a big deal.